ipv8/REST/attestation_endpoint.py

```diff
@@ -5,8 +5,8 @@
 from twisted.internet.defer import Deferred, inlineCallbacks, returnValue
 from twisted.web import resource
 
-from ..attestation.identity.community import IdentityCommunity
-from ..attestation.wallet.community import AttestationCommunity
+from ipv8.attestation.identity.community import IdentityCommunity
```

Comment:
This will break when imported from Tribler. We need relative imports.

Reply:
Yeah, I think I mentioned something about this on Slack. I used absolute imports instead of relative ones here deliberately. The reason for that is because for some reason, this time round, https://github.com/DanGraur/py-ipv8/blob/da8d4941247ad82905e3006c3e11b3a7606a9063/ipv8/REST/attestation_endpoint.py#L24 and https://github.com/DanGraur/py-ipv8/blob/da8d4941247ad82905e3006c3e11b3a7606a9063/ipv8/REST/attestation_endpoint.py#L25 would both return empty lists, which would break the `AttestationEndpoint` later on. The built-in function `isinstance` checks for the path of the packages and if I remember correctly, due to the relative import in the `attestation_endpoint.py` module for both `IdentityCommunity` and `AttestationCommunity`, the packages would differ when being tested by Jenkins on the Linux platform (I'm not exactly sure, but the packages would have something like `linuxunittest.attestation.identity.community` or something relatively similar). The paths for the classes of the overlays in the `session.overlays` list would remain constant across all of the platforms on which it was being tested (and if I remember correctly, they were the expected ones).

Reply:
We'll need a solution without absolute imports though. Using absolute imports will simply not run: this is a show stopper.

Reply:
Ok, I'll look into a solution for this which doesn't require absolute imports.